core/ecschema-rpc/common/src/ECSchemaRpcInterface.ts

```diff
@@ -13,7 +13,7 @@ import { SchemaKeyProps, SchemaProps } from "@bentley/ecschema-metadata";
  */
 export abstract class ECSchemaRpcInterface extends RpcInterface {
   /** The version of the RPC Interface. */
-  public static version = "1.0.0";
+  public static version = "1.1.0";
```

Comment:
From the README:
```
## Non-Zero Major Versions (released)

- Change in major version indicates a breaking change
- Change in minor version indicates a method was added
- Change in patch indicates a fix not affecting compatibility was made
```

You have changed the return type of an existing function. That is a breaking change requiring the major version to be incremented.

Reply:
Is anyone using this RpcInterface yet? If so consider deprecating and leaving unchanged the old method(s), and adding new method(s).
```
Semantic versioning can be used to communicate what happened.
However, the goal is usually to maintain compatibility as long as it is feasible.
In this case, compatibility means the ability of an older client to talk to a newer server.
Strategies that help maintain this type of compatibility include:

- Don't remove or modify existing methods
- Deprecate methods rather than removing them
- Only add new methods
- Don't add new methods until you are prepared to support them
- Develop new methods separately (see WipRpcInterface.ts) and only move in when ready for long term support
- If compatibility must be broken for an important performance or functionality reason, try to batch other breaking changes together to minimize the total impact on your consumers
```

Reply:
the GPB currently has it deployed in DEV, but aside from that no, it's limited to internal uses so far. I can bump the major version up in that case?

Reply:
That sounds safe.

Reply:
alright, I bumped the version up to 2.0.0